pyseq.py

```diff
@@ -80,6 +83,14 @@
 warnings.simplefilter('always', DeprecationWarning)
 
 
+def _natural_key(x):
```

Comment:
You're right I had tried it on some stereo stuff without success.  But I think presorting the list of files, or seqs, by whatever the pattern is get you faster results than having to check for an items membership in every previously found seq.  Though i can probably find a different sort key that would work for groups.